src/home/home.scss

```diff
@@ -99,6 +99,10 @@
     }
 
     &.launches {
+      > .heading1 {
+        -webkit-text-fill-color: $Secondary02;
```

Comment:
does it not work to set `background` instead of `-webkit-text-fill-color`?  It is, after all, `background` that you are supposedly overriding here.

Reply:
No, it does not